EIPS/bigint_modexp.md

```diff
@@ -0,0 +1,83 @@
+### Parameters
+
+* `GQUADDIVISOR: 100`
+
+### Specification
+
+At address 0x00......05, add a precompile that expects input in the following format:
+
+    <length_of_BASE> <length_of_EXPONENT> <length_of_MODULUS> <BASE> <EXPONENT> <MODULUS>
+    
+Where every length is a 32-byte left-padded integer representing the number of bytes to be taken up by the next value. Call data is assumed to be infinitely right-padded with zero bytes, and excess data is ignored. Consumes `floor(max(length_of_MODULUS, length_of_BASE) ** 2 * max(ADJUSTED_EXPONENT_LENGTH, 1) / GQUADDIVISOR)` gas, and if there is enough gas, returns an output `(BASE**EXPONENT) % MODULUS` as a byte array with the same length as the modulus.
+
+`ADJUSTED_EXPONENT_LENGTH` is defined as follows.
+
+* If `length_of_EXPONENT <= 256`, and all bits in `EXPONENT` are 0, return 0
```

Comment:
In this case, the specification literally checks the first 256 bytes (= 2048 bits) for non-zero bits.  I think this is not intended.